packages/modelviewer.dev/data/docs.json

```diff
@@ -1089,5 +1104,35 @@
         ]
       }
     ]
+  },
+  {
+    "Title": "PostProcessing Effects",
+    "htmlName": "postprocessing",
+    "Attributes": [
```

Comment:
I would want the same docs style for the effects, plus the examples aren't listing all the parameters and how they affect the look of the final postprocessing. I was thinking it could be just another docs page that is referenced from the main docs page and from the examples, with the same structure as the main docs page.